### PR TITLE
Included forth PR instructions

### PR DESCRIPTION
**Pull request by RafaelBatman55**
## Description

- Included forth PR description in the end of the document following #417 instructions.

### CODE REVIEW

File: docs/ONBOARDING.md [hunk 1]
```diff
@@ -68,6 +69,15 @@ Once you are part of the twindle team, please join us here to keep in touch.
 
 
 ---
+
+5. Do the forth PR
```
Comment (piratelicorne) [inline docs/ONBOARDING.md:73]: change it to 'fourth'

File: docs/ONBOARDING.md [hunk 2]
```diff
@@ -68,6 +69,15 @@ Once you are part of the twindle team, please join us here to keep in touch.
...
+<br>
+
+**What is the forth PR about?**
```
Comment (piratelicorne) [inline docs/ONBOARDING.md:76]: change it to 'fourth'

File: docs/ONBOARDING.md (hunk 1 above)
Comment (RafaelBatman55) [reply docs/ONBOARDING.md]: Thank you!

File: docs/ONBOARDING.md (hunk 2 above)
Comment (RafaelBatman55) [reply docs/ONBOARDING.md]: Thank you!

**Review by shekhar10feb — APPROVED** [review]

**Review by piratelicorne — APPROVED** [review]

**Comment by piratelicorne** [comment]
Approved :)

**Review by ni3-kdm — APPROVED** [review]
Looks fine!

**Pull request merged**